### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by baskren** [CoreCompat/libgdiplus-packaging#4]
I've added this package to my Xamarin.Mac project without success (calls to instantiate a System.Drawing.Drawing2D.Matrix fail).  I'm guessing I'm missing something.  Besides adding the Nuget Package, are there other steps required to use this?

**Comment by jamescoverdale** [comment]
@baskren Did you solve your issue? Currently seeing the same thing, have installed the nuget package but still getting the same errors.

**Comment by baskren** [comment]
No, never did.  Tried A LOT of approaches but nothing worked.  Ended up writing the pieces I needed myself and substituting that code.

**Comment by jamescoverdale** [comment]
Not sure if its an issue with my version of Visual Studio for MacOS (8.7.2), but I had to manually add the native references from `packages/runtime.osx.10.10-x64.CoreCompat.System.Drawing.5.8.64/runtimes/osx-x64/native` to my project.

![Screenshot 2020-09-03 at 12 45 18](https://user-images.githubusercontent.com/1148419/92110943-6e2ca480-ede3-11ea-9787-c10f341ca083.png)
